EIPS/eip-2935.md

```diff
@@ -234,6 +202,12 @@ def process_block_hash_history(block: Block, state: State):
 
 The first option is recommended until the Verkle fork, to stay consistent with [EIP-4788](./eip-4788.md) and to issues for misconfigured networks where this EIP is activated but history contract hasn't been deployed. The recommendation may be reconsidered at the Verkle fork if filtering the system contract code chunks is deemed too complex.
 
+### Size of ring buffers
+
+The ring buffer data structure is sized to hold 8191 hashes. In other system contracts a prime ring buffer size is chosen in because using a prime as the modulus ensures that no value is overwritten until the entire ring buffer has been saturated and thereafter, each value will be updated once per iteration, regardless of if some slot are missing or the slot time changes. However, in this EIP the block number is the value in the modulo operation and it only ever increases by 1 each iteration. Which means we can be confident that the ring buffer will always remain saturated.
```

Comment:
```suggestion
The ring buffer data structure is sized to hold 8191 hashes. In other system contracts a prime ring buffer size is chosen in because using a prime modulus ensures that the entire ring buffer gets saturated. However, in this EIP the block number only ever increases by 1 each iteration fully saturating the buffer before wrapping around.
```